generators/app/templates/my-awesome-site/gulpfile.babel

```diff
@@ -74,17 +74,19 @@ gulp.task('css', () => {
 <% if (includeSass || includeScss) { -%>
 // Compile scss to css.
 gulp.task('scss', () => {
+//This will check for sass in all pages of the site, incase you use them
+//TODO: Move all Jekyll setup in a folder, build to another folder
 <% if (includeSass) { -%>
-    return gulp.src('sass/main.sass')
+    return gulp.src(['**/css/*.sass','!_site']])
 <% } -%>
 <% if (includeScss) { -%>
-    return gulp.src('scss/main.scss')
+    return gulp.src(['**/css/*.scss','!_site'])
```

Comment:
I think you meant `'**/sass/*.sass'` and `'**/scss/*.scss'`, because if the user choose `sass`/`scss` when generate the project he gets a directory with this name.

Also, this task compile the `sass`/`scss` files from the `node_modules` directory.